src/pages/[...slug].tsx

```diff
@@ -142,6 +147,8 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
     }
   }
 
+  const crowdinContributors = getCrowdinContributors(mdPath, locale as Lang)
```

Comment:
@pettinarip This is also used in the dev docs ([example](https://ethereum.org/es/developers/docs/blocks/)).

Reply:
But that appears to be it... can add a conditional for these two layouts to help prevent unneeded work for the other layouts

Reply:
I think it would make a good impact since excluding the docs & tutorials, we still have 86 pages to build that are not going to use the result of this function.

If you get the total, it would be `86*60 langs = 5160` pages that gonna benefit from doing it.

Reply:
@pettinarip Added a conditional to only fetch contributors for the `docs` and `tutorials` template... These are currently the only places this component it consumed, and although it's a synchronous call, with thousands of pages figured it doesn't hurt to trim operations where we can. 

Also, through this realized we aren't typing out Layouts... added PR #137 to implement a `Layout` type based on the `layoutMapping` in `[...slug]`